src/SignaturePad.InkPresenter.Shared/InkPresenter.cs

```diff
@@ -9,6 +9,13 @@
 using NativeColor = Android.Graphics.Color;
 using NativeImage = Android.Graphics.Bitmap;
 using NativePath = Android.Graphics.Path;
+#elif NET471
+using NativePath = System.Drawing.Drawing2D.GraphicsPath;
+using NativeRect = System.Drawing.RectangleF;
+using NativePoint = System.Drawing.Point;
+using NativeSize = System.Drawing.Size;
```

Comment:
Is it possible to use the floating point versions? `PointF`, `SizeF` with `RectangleF`?

Reply:
using NativeRect = System.Drawing.RectangleF;
using NativePoint = System.Windows.Input.StylusPoint;